windowsforms/Sudoku/VB/ReadMe.MD

```diff
@@ -0,0 +1,42 @@
+languages:
+    - vb
+products:
+    - dotnet-core
+    - windows
+page_type: sample
+name: ".NET Core WinForms Sudoku"
+urlFragment: "Sudoku-vb"
+description: "Sudoku is a number puzzle game, played on a nine by nine cell grid"
+# Sudoku
+## Requires
+- Visual Studio 2019 Version 16.5.0 Preview 1.0 or later
```

Comment:
I think .NET Core 3.1 is released with Visual Studio 2019 version 16.4. so this example doesn't really require 16.5 ?

Reply:
It is the first time you can view and edit VB WinForms so it is debatable. I am good either way,

Reply:
Oh, I didn't know that the designer for .NET Core WinForms is even released. Then, we can see @mairaw & @BillWagner opinions.

Reply:
https://devblogs.microsoft.com/dotnet/updates-to-net-core-windows-forms-designer-in-visual-studio-16-5-preview-1/

> You need to enable the designer in Visual Studio. Go to Tools > Options > Environment > Preview Features and select the Use the preview Windows Forms designer for .NET Core apps option.

I think these instructions would be helpful to add in this readme.

Reply:
btw, Are you migrating this from an old system ? or wrote it entirely ?

Reply:
It is the original files from the archive, with the project file converted manually for Core 3.1, and warning/suggestions fixed (things like Make Readonly and removal of ByVal. Plus format changes required because of the default font change and Adding the ICO file

Reply:
If you don't enable it the first time you try, it warns you and takes you to the setting. So very discoverable. What is missing in https://devblogs.microsoft.com/dotnet/updates-to-net-core-windows-forms-designer-in-visual-studio-16-5-preview-1/ is VB but I hope that is fixed in preview-2.